### PR TITLE
Bug 1116601 - More intuitive optioncollection API

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

**Comment by maurodoglio** [comment]
See comment on bug 1116601

**Comment by maurodoglio** [comment]
It looks good to me, I'm just wondering why you didn't use rest_framework for this endpoint?
I know it's a very custom one, but for example since your view is not registered on the default router, it won't be listed on treeherder.mozilla.org/api/

**Comment by wlach** [comment]
We can use rest_framework I guess, I didn't know about the api listing thing so didn't see the point. We should probably add a project rest_framework endpoint I guess in that case?

**Comment by maurodoglio** [comment]
I see you just pushed some code to do that :smile:

File: treeherder/webapp/api/refdata.py [hunk 1]
```diff
@@ -95,6 +95,21 @@ class OptionCollectionViewSet(viewsets.ReadOnlyModelViewSet):
     model = models.OptionCollection
 
 
+class OptionCollectionHashViewSet(viewsets.ViewSet):
+    """ViewSet for the virtual OptionCollectionHash model"""
+
+    def list(self, request):
+        rdm = RefDataManager()
```
Comment (maurodoglio) [inline treeherder/webapp/api/refdata.py]: can you please explicitly disconnect rdm in a finally clause?

**Comment by wlach** [comment]
Merged!

**Pull request closed without merging**